_generator/types.yaml

```diff
@@ -325,16 +325,16 @@ types:
     anchor: resource
   resourcedatacoproductofspacedataobjectdatapersondata:
     file: resources.md
-    anchor: resource-data
+    anchor: data
   spacedata:
     file: resources.md
-    anchor: space-resource-data
+    anchor: spacedata
```

Comment:
the anchors should stay kebab-case (words separated by dash). Add `Title` to this type and to all other types missing the title

Reply:
types.yaml reverted and generator re-run as discussed in refinement meeting.

Reply:
While `types.yaml` is a useful heuristic, the main concern should be rendering of the markdown, hence not all entries here will be hyphenated.

Reply:
will add this info to generator documentation so its clear